src/modules/household/components/elements/AddToHouseholdButton.tsx

```diff
@@ -76,8 +78,9 @@ const AddToHouseholdButton = ({
         </Button>
       </ButtonTooltipContainer>
       {renderFormDialog({
-        title: <>Enroll {clientName}</>,
+        title: <>Enroll {clientBriefName(client)}</>,
         submitButtonText: `Enroll`,
+        preFormComponent: <ClientAlertEnrollmentWrapper client={client} />,
```

Comment:
Do we really need this wrapper component? There are a lot of small components here and I wonder if some of them can be consolidated. The logic is already encapsulated in the hook, so we might as well just call that hook in this component, I think.

Eg:

```
const { clientAlerts } = useClientAlerts({ client }, false);

...

{renderFormDialog({
        title: <>Enroll {clientName}</>,
        title: <>Enroll {clientBriefName(client)}</>,
        submitButtonText: `Enroll`,
        preFormComponent: {clientAlerts.length > 0 && <ClientAlertStack clientAlerts={clientAlerts} />},

```